### PR TITLE
Fix :attribute id's-fix

### PR DESCRIPTION
**Pull request by RamaChandiniDevarapalli**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/components/AsideLinks.tsx [hunk 1]
```diff
@@ -14,12 +14,12 @@ function AsideLinks({ serviceLink, docsLink }: AsideLinksProps): JSX.Element {
       <GapAdjustment>Helpful links</GapAdjustment>
       {serviceLink && (
         <HyperLinkColor>
-          <ExternalLink link={serviceLink} text="See the code" />
+          <ExternalLink testId="code-link" link={serviceLink} text="See the code" />
```
Comment (thdlam) [inline apps/tenant-management-webapp/src/app/components/AsideLinks.tsx:17]: Should this be data-testid?

File: apps/tenant-management-webapp/src/app/components/AsideLinks.tsx [hunk 2]
```diff
@@ -14,12 +14,12 @@ function AsideLinks({ serviceLink, docsLink }: AsideLinksProps): JSX.Element {
...
         </HyperLinkColor>
       )}
       {docsLink && (
         <HyperLinkColor>
-          <ExternalLink link={docsLink} text="Read the API docs" />
+          <ExternalLink link={docsLink} testId="docs-link" text="Read the API docs" />
```
Comment (thdlam) [inline apps/tenant-management-webapp/src/app/components/AsideLinks.tsx:22]: Should this be data-testid

File: apps/tenant-management-webapp/src/app/components/AsideLinks.tsx (hunk 1 above)
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/components/AsideLinks.tsx:17]: The property added to the external link is testId

**Review by thdlam — APPROVED** [review]

File: apps/tenant-management-webapp/src/app/components/AsideLinks.tsx (hunk 1 above)
Comment (solittlework) [reply apps/tenant-management-webapp/src/app/components/AsideLinks.tsx:17]: I just checked the code and it looks the ExternalLink has the "testId" property.

File: apps/tenant-management-webapp/src/app/components/AsideLinks.tsx (hunk 2 above)
Comment (RamaChandiniDevarapalli) [reply apps/tenant-management-webapp/src/app/components/AsideLinks.tsx:22]: interface ExternalLinkProps {
  text: string;
  link: string;
  testId?: string;
}
testId is the property in ExternalLink

**Pull request merged**